x-pack/test/api_integration/services/index.ts

```diff
@@ -43,4 +44,5 @@ export const services = {
   usageAPI: UsageAPIProvider,
   ml: MachineLearningProvider,
   ingestManager: IngestManagerProvider,
+  resolverGenerator: ResolverGeneratorProvider,
```

Comment:
Adding the resolver generator as a service so it can be used in API integration tests.